.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 /dist
 /build
 /lib
+.vscode
```

Comment:
Done. I didn't know there was a setting for global .gitignore - that's really useful.